1xx-enable-ssl-for-kafka-bridge.md

```diff
@@ -0,0 +1,137 @@
+# Add support for TLS/SSL on the HTTP interface 
+
+This proposal adds an option to enable TLS encryption on HTTP interface so that requests from clients to the Kafka Bridge can be sent with encryption.
+
+## Current situation
+
+Strimzi does not support TLS/SSL for HTTP Bridge, so client connections to HTTP Bridge are not encrypted at all. There is a single listener for all the endpoints that are used for Kafka operations as well as admin operations such as healthcheck and monitoring and it runs on port 8080 by default.
+
+## Motivation
+
+The connection between HTTP Bridge and Kafka cluster can be secured however, currently there is no support for securing the connections to HTTP Bridge from clients. The current recommendation is to use firewalls or API gateways to secure the client connections. This requires users to make additional configurations and does not provide the best user experience. 
+
+Moreover, some of the endpoints are for internal use only, such as `/healthy`, `/ready` and `/metrics` but they are exposed on the same listener and the same port as the other endpoints that are used by external clients. This means the endpoints for internal use only are also exposed to external clients. And if SSL is enabled, it will introduce more operational complexity as these internal endpoints would need to be configured with TLS and certificates.
+
+## Proposal
+This proposal also adds new configurations to enable SSL server and specify locations of certificate and key files in PEM format so that client connections to HTTP Bridge can be encrypted. The new configurations will be added to the current `http` prefixed configurations:
+- http.ssl.enable
+- http.ssl.keystore.location
+- http.ssl.keystore.key.location
```

Comment:
What is the difference between `http.ssl.keystore.location` and `http.ssl.keystore.key.location`? Also, should we:
* Have some type field? What formats will we support? You seem to mention PEM only. But given how common JKS/PKCS12 is in Java land, we should be at least ready to support them. They would also need a password field I guess.
* Have some field to pass the keystore public and private key directly? E.g. `http.ssl.keystore.key` and `http.ssl.keystore.certificate.chain`?
* Would it make more sense to have the options named as `https`? E.g. `https.enable`?